src/pages/Homepage/Homepage.vue

```diff
@@ -72,26 +69,33 @@ export default {
 		query: activePageQuery,
 		preFetch: true,
 		result({ data }) {
-			// isIwdActive used to be set based on an experiment query
-			// if something like this is to be implemented in the future it
-			// should be based on contentful, like isWrdActive.
-			this.isIwdActive = false;
-
-			// determine if take-over message content is active
-			const contentSetting = _get(data, 'contentful.entries.items', []).find(item => item.fields.key === 'ui-homepage-top'); // eslint-disable-line max-len
-			if (!contentSetting || !contentSetting.fields) {
-				this.takeOverActive = false;
-			} else {
+			// determine if take-over setting is active
+			const contentSetting = _get(data, 'contentful.entries.items', []).find(item => item.fields.key === 'ui-homepage-takeover'); // eslint-disable-line max-len
+			if (_get(contentSetting, 'fields')) {
 				const isContentEnabled = settingEnabled(
 					contentSetting.fields,
 					'active',
 					'startDate',
 					'endDate'
 				);
 				if (isContentEnabled) {
+					const takeOverKey = _get(contentSetting, 'fields.dataObject.takeOverKey');
```

Comment:
I think this is a cleaner way to pick 1 homepage takeover design based on a single setting in contentful